Grub2summit (#339)
3mdeb/news-and-ideas: blog/content/post/2020-11-02-GRUB2mini-summit.md

---

> title: GRUB2 mini–summit opening

Reviewer: ```suggestion
title: GRUB mini–summit opening
```
As DK suggested

abstract: 'GRUB mini–summit 2020. This year we cannot miss this opportunity to meet again

> and face the new challenges of GRUB/GRUB2. So,dear reader, feel invited to look

Reviewer: ```suggestion
and face the new challenges of GRUB/GRUB2. So, dear reader, feel invited to look
```

at GRUB2 with a magnifying glass.'
cover: /covers/GRUB.jpeg
author: kamila.banecka
layout: post
published: true
date: 2020-11-02
archives: "2020"

tags:
  - grub2
  - bootloader
  - trenchboot
categories:
  - Firmware
  - OS Dev
  - Security

---
#Intro

>  Sometimes we should to stop and look at the tools without which it would be

Reviewer: ```suggestion
 Sometimes we should stop and look at the tools without which it would be
```
Is this what you meant? @KBanecka

>  difficult to even imagine everyday work. Some of them are so essential, that to

Reviewer: ```suggestion
 difficult to even imagine everyday work. Some of them are so essential, that
```

>  avoid them means to develop a wheel once again in the history. Such a tool in

Reviewer: ```suggestion
 avoiding them means to develop a wheel once again in history. Such a tool in
```

>  the world of firmware – along with keyboard, fridge and linux – is a dwarf that

Reviewer: ```suggestion
 the world of firmware – along with keyboard, fridge and Linux – is a dwarf that
```

 became synonymous with the word bootloader. And by dwarf I mean of course GRUB.
 Last December, we've met with our friend, Daniel Kiper, GRUB upstream
 maintainer and TrenchBoot technical leader inside Oracle. This meeting resulted
 in organization of GRUB minisummit 2019, because we wanted to understand a
 vision of both, GRUB/GRUB2 community and commercial perspective. All the event
 was fruitful and grounded some common path that will be slightly summarized
 below. This year we cannot miss this opportunity to meet again and face the

>  new challenges of GRUB/GRUB2. So,dear reader, feel invited to look at GRUB2

Reviewer: ```suggestion
 new challenges of GRUB/GRUB2. So, dear reader, feel invited to look at GRUB2
```

 with a magnifying glass.

  **We are starting tommorow**. No axe needed.

![tommorow schedule](/img/Grub.jpg)

Piotr Król, founder of 3mdeb will open the event introducing guests,
contributors, the history of mini-summit and the main goals of the meeting.
After the introduction Michał Żygowski, TrenchBoot contributor from 3mdeb, will
start the first talk dedicated to the GRUB network stack. It will be compared

> with other solutions (such as iPXE in terms of performance), because sometimes

Reviewer: ```suggestion
with other solutions (such as iPXE in terms of performance). Sometimes
```

in order to recover used operating system we need to boot another operating
system, ().e.g live image to perform recovery operations), where the easiest
method to do so is to boot from network. The last talk, performed by our guest,
Daniel Kiper will bring the inner view for the current GRUB2 project status.
Daniel will tell more about what was done during the recent year, what is
performed now and what are planned priorities for the nearest future. Talks will
end with the opening questions for the Ask Me Anything session, during which the

> participants can question related topics and foster discussion.

Reviewer: ```suggestion
audience can question related topics and foster discussion.
```

### Where to link the event?

You will find us on our YouTube 3mdeb channel. All you need to do is joining our
live [here](https://www.youtube.com/channel/UC_djHbyjuJvhVjfT18nyqmQ/live ) in
an appropriate time presented above. **We are starting GRUB mini-summit tommorow
at 04:00 PM CET (UTC+1:00)**.

The next mini-summit days will gather interesting topics concerned on AMD
TrenchBoot, RISC-V support in GRUB2, the Firmware and Bootloader log
specification, license issues and more. Next Tuesday, our special guest from
9elements will introduce us with the XHCI Support in GRUB2. Worthy to wait for,
the full agenda of the next meetings will be updated here and on our SM.

> ### Whom will you meet?

Reviewer: ```suggestion
### Whom will you meet?

```
